src/EA.Weee.Web/Areas/Aatf/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -120,6 +126,11 @@ public async Task<ActionResult> CreateEvidenceNote(EditEvidenceNoteViewModel vie
         {
             using (var client = apiClient())
             {
+                if (viewModel != null && viewModel.Action == ActionEnum.CopyAndPaste)
```

Comment:
probably wouldnt bother doing the null check for the viewmodel here... if thats null we have bigger problems to worry about, same for the edit

Reply:
Null checks removed, force of habit